client/my-sites/customer-home/cards/education/educational-content/index.jsx

```diff
@@ -26,32 +26,30 @@ const EducationalContent = ( { title, description, links, illustration } ) => {
 					{ description }
 				</p>
 				<div className="educational-content__links">
-					{ links.map( ( { postId, url, externalLink, text, icon, tracksEvent, statsName } ) => (
-						<div
-							className={ classnames( 'educational-content__link', {
-								'is-external-link': externalLink,
-							} ) }
-							key={ url }
-						>
-							{ icon && <Gridicon icon={ icon } size={ 18 } /> }
-							{ postId && (
-								<InlineSupportLink
-									supportPostId={ postId }
-									supportLink={ url }
-									showIcon={ false }
-									text={ text }
-									tracksEvent={ tracksEvent }
-									statsGroup="calypso_customer_home"
-									statsName={ statsName }
-								/>
-							) }
-							{ externalLink && (
-								<ExternalLink href={ url } icon>
-									{ text }
-								</ExternalLink>
-							) }
-						</div>
-					) ) }
+					{ links.map(
+						( { postId, url, calypsoLink, externalLink, text, icon, tracksEvent, statsName } ) => (
+							<div className={ classnames( 'educational-content__link' ) } key={ url }>
```

Comment:
Now that we don't add the `is-external-link` class this could be simplified to:
```suggestion
							<div className="educational-content__link" key={ url }>
```